pkgs/applications/networking/cluster/nomad-device-nvidia/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "nomad-device-nvidia";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = pname;
```

```suggestion
    repo = "nomad-device-nvidia";
```

`repo = pname` is discouraged.